Dev

bashun98 opened the pull request:
Вова, проверяй)

bashun98 committed:
fix readme

bashun98 committed:
Merge pull request #2 from Auctix/feature/eugene

fix readme

ShagMichail committed:
edit README

ShagMichail committed:
Первые изменения: добавление экрана загрузки, начала проектирования основного экрана

ShagMichail committed:
create UITabBarController

ShagMichail committed:
убрал ненужные файлы

ShagMichail committed:
Merge remote-tracking branch 'refs/remotes/origin/dev'

Conflicts:
	README.md

ShagMichail committed:
Изменение TabBar

ShagMichail committed:
Исправил недочеты ментора и распределил по папочкам

ShagMichail committed:
Добавил файлы для коллекции домашнего экрана

ShagMichail committed:
Изменения колекции на 23:45 22.10

ShagMichail committed:
Изменение колекции 00:42 23:10

ShagMichail committed:
Добавление другой модели и изменение ячейки

KamnevVladimir committed:
Пример

ShagMichail committed:
Исправил некоторые недочеты, сделал добавление новых выставок, добавил тень на таббар

ShagMichail committed:
Добавление контролера для таблицы товаров и действия на кнопку

ShagMichail committed:
Изменение коллекции

ShagMichail committed:
Кнопка колекции активна, не исправен target

ShagMichail committed:
Поменял место расположение таргета

ShagMichail committed:
Новый таргет

ShagMichail committed:
Последня права таргета 25.10

ShagMichail committed:
Добавление ячеек таблицы, попытки добавление градиента

ShagMichail committed:
Добавил экран с продуктами и поправил таблицу продуктов

ShagMichail committed:
Проверка связи

ShagMichail committed:
Поменял название папки

ShagMichail committed:
Delete HomeButtonTabViewController.swift

ShagMichail committed:
Delete Auctix/Extensions directory

ShagMichail committed:
Delete ExhibitionCell.swift

ShagMichail committed:
Delete ExhibitionManager.swift

ShagMichail committed:
Delete logo.jpg

ShagMichail committed:
добавление новой папки

ShagMichail committed:
Merge remote-tracking branch 'refs/remotes/origin/dev'

ShagMichail committed:
Delete Auctix/TableGoods directory

ShagMichail committed:
Delete Auctix/TableGoods directory

ShagMichail committed:
Merge remote-tracking branch 'refs/remotes/origin/feature/misha/navBar'

bashun98 committed:
add onboarding

bashun98 committed:
add ExhibitionTableViewCell

bashun98 committed:
add list of exhibitions

bashun98 committed:
add ext for fonts(still not working)

ShagMichail committed:
Merge remote-tracking branch 'refs/remotes/origin/dev'

ShagMichail committed:
Поправил перемещение домашней коллекции

ShagMichail committed:
Merge pull request #6 from Auctix/feature/misha/navBar

Feature/misha/nav bar

ShagMichail committed:
Начал переделывать страницу продукта

ShagMichail committed:
Поправил связь

ShagMichail committed:
Поменял страницу продукта

ShagMichail committed:
Сделал страницу продукта со ставкой без фотографии и изменения фактической цены

ShagMichail committed:
Merge pull request #7 from Auctix/feature/misha/navBar

Feature/misha/nav bar

ShagMichail committed:
Добавил фотографию и изменил цвет надписей

IvanTuly committed:
Добавление страницы аккаунта

IvanTuly committed:
Merge pull request #8 from Auctix/feature/ivan/account

Добавление страницы аккаунта

ShagMichail committed:
Merge pull request #9 from Auctix/feature/misha/navBar

Добавил фотографию и изменил цвет надписей

bashun98 committed:
add delegate for header

bashun98 committed:
fix exhibition labels

bashun98 committed:
add picker view for sorting

bashun98 committed:
add picker view with sorting (not ready)

bashun98 committed:
add sorting for arrays

bashun98 committed:
add arrow triangle to sort button

bashun98 committed:
update picker view height

bashun98 committed:
add country sorting

bashun98 committed:
Merge pull request #11 from Auctix/feature/eugene/fixlist

Feature/eugene/fixlist

KamnevVladimir commented on Auctix/Account/AccountButtonController.swift:
```diff
@@ -0,0 +1,199 @@
+        table.translatesAutoresizingMaskIntoConstraints = false
+        table.alwaysBounceVertical = false
+        
+        table.register(SwitchTableViewCell.self,
```
- [ ] Настройку таблицы (делегаты, регистрации ячеек) вынесите в отдельный метод

KamnevVladimir commented on Auctix/Account/AccountButtonController.swift:
```diff
@@ -0,0 +1,199 @@
+        setupImage()
+        view.addSubview(image)
+        setupUserNameLabel()
+        view.addSubview(label)
```
- [ ] лучше чистый viewDidLoad держать, вынесите в setupView

KamnevVladimir commented on Auctix/Account/AccountButtonController.swift:
```diff
@@ -0,0 +1,199 @@
+        var constraints = [NSLayoutConstraint]()
+        //add
+        constraints.append(tableView.leadingAnchor.constraint(
+            equalTo: view.safeAreaLayoutGuide.leadingAnchor))
```
Не обяз

- Так вообще не пишут, все пишут сразу в NSLayoutConstraint.activate([ констрейнты ])

KamnevVladimir commented on Auctix/Account/AccountButtonController.swift:
```diff
@@ -0,0 +1,199 @@
+
+import UIKit
+
+class AccountButtonTabViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
```
- Важно

- [ ] Поменять нейминг, лучше, например, AccountViewController

KamnevVladimir commented on Auctix/Account/AccountButtonController.swift:
```diff
@@ -0,0 +1,199 @@
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let model = models[indexPath.section].options[indexPath.row]
+        
+        switch model.self {
```
хорош, возьму себе на заметку

KamnevVladimir commented on Auctix/Account/AccountButtonController.swift:
```diff
@@ -0,0 +1,199 @@
+        }
+    }
+    //MARK: настраиваем наполнение каждой строки таблицы настроек
+    func configure() {
```
- на будущее

Это ближе к makeState, makeModel, setupModel

KamnevVladimir commented on Auctix/Account/Data.swift:
```diff
@@ -0,0 +1,36 @@
+
+enum SettingOptionType {
+    case staticCell(model: SettingsOption)
+    case switchCell(model: SettingsSwitchOption)
```
ассоциативные - хорош

KamnevVladimir commented on Auctix/Account/Data.swift:
```diff
@@ -0,0 +1,36 @@
+struct SettingsOption {
+    let title: String
+    let icon: UIImage?
+    let iconBackgroundColor: UIColor
```
iconBackgroundColor -> iconBackColor/backColor

KamnevVladimir commented on Auctix/Account/TableViewCell/SettingTableViewCell.swift:
```diff
@@ -0,0 +1,80 @@
+
+class SettingTableViewCell: UITableViewCell {
+
+    static let identifier = "SettingTableViewCell"
```
- обяз

- [ ] Вынести в расширение, скидывал в чат

KamnevVladimir commented on Auctix/Account/TableViewCell/SettingTableViewCell.swift:
```diff
@@ -0,0 +1,80 @@
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        contentView.addSubview(label)
+        contentView.addSubview(iconContainer)
+        iconContainer.addSubview(iconImageView)
```
- обяз

- [ ] В Сетапы

KamnevVladimir commented on Auctix/Account/TableViewCell/SettingTableViewCell.swift:
```diff
@@ -0,0 +1,80 @@
+        accessoryType = .disclosureIndicator
+    }
+    
+    required init?(coder: NSCoder) {
```
return nil

KamnevVladimir commented on Auctix/Account/TableViewCell/SettingTableViewCell.swift:
```diff
@@ -0,0 +1,80 @@
+        iconContainer.frame = CGRect(x: 15, y: 6, width: size, height: size)
+        
+        let imageSize = size/1.5
+        iconImageView.frame = CGRect(x: (size-imageSize)/2, y: (size-imageSize)/2, width: imageSize, height: imageSize)
```
- обяз
в методы

- не обяз
Цифры и пр в struct Constants { static let a, static let b }

KamnevVladimir commented on Auctix/Account/TableViewCell/SwitchTableViewCell.swift:
```diff
@@ -0,0 +1,96 @@
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        contentView.addSubview(label)
+        contentView.addSubview(iconContainer)
+        contentView.addSubview(mySwitch)
```
тоде самое

KamnevVladimir commented on Auctix/Assets.xcassets/AppIcon.appiconset/Contents.json:
```diff
@@ -1,91 +1,115 @@
       "size" : "20x20"
     },
     {
+      "filename" : "Icon-App-29x29@1x.png",
```
- [ ] Нейминги у картинок тоже соблюдайте
Пример

В папке icons лежит папка city, а там согласно неймингу лежит icons_city_route

Нижнее подчеркивание - как слэш в URL 
End Point - имя файла

Сейчас можно не править, но следующие уже с неймингом заливайте

KamnevVladimir commented on Auctix/Bid/BidButtonController.swift:
```diff
@@ -0,0 +1,37 @@
+
+import UIKit
+
+class BidButtonTabViewController: UIViewController {
```
Нейминг

 BidViewController

KamnevVladimir commented on Auctix/Bid/BidButtonController.swift:
```diff
@@ -0,0 +1,37 @@
+
+class BidButtonTabViewController: UIViewController {
+
+    let label = UILabel()
```
UI вообще весь сделайте private

KamnevVladimir commented on Auctix/Bid/BidButtonController.swift:
```diff
@@ -0,0 +1,37 @@
+      
+        navigationController?.navigationBar.isHidden = true
+        label.text = "Bid"
+        label.backgroundColor = .red
```
в методы

KamnevVladimir commented on Auctix/Bid/BidButtonController.swift:
```diff
@@ -0,0 +1,37 @@
+    override func viewWillLayoutSubviews() {
+        super.viewWillLayoutSubviews()
+        setupLayuot()
+        
```
лишние отступы везде убрать

KamnevVladimir commented on Auctix/Color/Ext+UIColor.swift:
```diff
@@ -0,0 +1,33 @@
+    }
+    public class var lightCornflowerBlue  : UIColor {
+        return UIColor(named: "Light Cornfiower Blue")   ?? UIColor.white
+    }
```
оки

KamnevVladimir commented on Auctix/ExhibitionCell/Entities/Cell + Data/Exhibition.swift:
```diff
@@ -0,0 +1,17 @@
+struct Exhibition{
+    let titleImg: UIImage
+    let title: String
+    let text_opis: String
```
не по свифтовские

textOpis

KamnevVladimir commented on Auctix/ExhibitionCell/Entities/Cell + Data/ExhibitionCell.swift:
```diff
@@ -0,0 +1,113 @@
+        imageExhib.addSubview(jumpButton)
+        //jumpButton.addTarget(self, action: #selector(didTabButton(sender:)), for: .touchUpInside)
+        
+    }
```
отступов лишних очень много

Совсем старые и ненужные комменты тоже лучше сразу стирай (если не нужны)

KamnevVladimir commented on Auctix/ExhibitionCell/Entities/Cell + Data/ExhibitionCell.swift:
```diff
@@ -0,0 +1,113 @@
+    static var nib  : UINib{
+        return UINib(nibName: identifire, bundle: nil)
+    }
+    
```
Правильно расширение, но не в правильном место

Его нужно кинуть на UITableViewCell и на UICollectionViewCell

ShagMichail replied:
Не понял, они же и так в этих файлах

KamnevVladimir commented on Auctix/Home/HomeButtonController.swift:
```diff
@@ -0,0 +1,224 @@
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        
```
уф

ShagMichail replied:
Если про отступы, то окей)

KamnevVladimir commented on Auctix/Home/HomeButtonController.swift:
```diff
@@ -0,0 +1,224 @@
+        
+        view.addSubview(titleLabel)
+        view.addSubview(nameLabel)
+        view.addSubview(searchTextField)
```
в метод

KamnevVladimir commented on Auctix/Home/HomeButtonController.swift:
```diff
@@ -0,0 +1,224 @@
+// обработчик нажатия кнопки на ячейке коллекции (открывает таблицу продуктов)
+extension HomeButtonTabViewController  {
+    
+    @objc func didTabButton(sender: UIButton) {
```
не обяз

Многие для удобства и более короткой строки пишут
@objc 
func didTapButton

KamnevVladimir commented on Auctix/Home/HomeButtonController.swift:
```diff
@@ -0,0 +1,224 @@
+            nameLabel.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -41),
+            nameLabel.topAnchor.constraint(equalTo: titleLabel.bottomAnchor, constant: 24),
+            
+            searchTextField.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 36),
```
36 всяко в константы можно

ShagMichail replied:
В чем удобство?

KamnevVladimir commented on Auctix/Home/HomeButtonController.swift:
```diff
@@ -0,0 +1,224 @@
+            .foregroundColor: UIColor.honeyYellow
+        ])
+        
+        let ctixWords = NSAttributedString(string: "UCTIX", attributes: [
```
aWord
bWord

KamnevVladimir commented on Auctix/Home/HomeButtonController.swift:
```diff
@@ -0,0 +1,224 @@
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, insetForSectionAt section: Int) -> UIEdgeInsets {
+        return .init(top: 20, left: 25, bottom: 50, right: 25)
```
прям тоже в 1 константу можно

KamnevVladimir commented on Auctix/List/ExhibitionTableViewCell.swift:
```diff
@@ -0,0 +1,108 @@
+
+class ExhibitionTableViewCell: UITableViewCell {
+    
+    static let identifier = "CustomTableViewCell"
```
когда перенесете в расширение, которое лежит в одной из ячеек уже, то удалите везде static let identifier

KamnevVladimir commented on Auctix/List/ExhibitionTableViewCell.swift:
```diff
@@ -0,0 +1,108 @@
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
```
return nil

KamnevVladimir commented on Auctix/List/ExhibitionTableViewCell.swift:
```diff
@@ -0,0 +1,108 @@
+        gradient.endPoint = CGPoint(x: 0.75, y: 0.5)
+        gradient.cornerRadius = layer.cornerRadius
+        gradient.transform = CATransform3DMakeAffineTransform(CGAffineTransform(a: -1, b: 0, c: 0, d: -5.8, tx: 1, ty: 3.4))
+        gradient.bounds = bounds.insetBy(dx: -0.5*bounds.size.width, dy: -0.5*bounds.size.height)
```
стиль свифта не забываем
-0.5 * bounds.size.width

KamnevVladimir commented on Auctix/List/ExhibitionTableViewCell.swift:
```diff
@@ -0,0 +1,108 @@
+    
+    private func setupLayout() {
+        myImage.frame = contentView.bounds.inset(by: UIEdgeInsets(top: 5, left: 12, bottom: 5, right: 12))
+        exhibitionName.snp.makeConstraints { make in
```
определитесь, что будете юзать

snapkit, pinlayout ?

KamnevVladimir commented on Auctix/List/ListButtonController.swift:
```diff
@@ -0,0 +1,160 @@
+import UIKit
+import SnapKit
+
+class ListButtonTabViewController: UIViewController {
```
нейминг

KamnevVladimir commented on Auctix/List/ListButtonController.swift:
```diff
@@ -0,0 +1,160 @@
+    private var sortLabel: String = " "
+    
+    let tableView: UITableView = {
+        let tableView = UITableView()
```
вся эта настройка в метод setupTableView

здесь останется просто let tableView = UITableView()

KamnevVladimir commented on Auctix/List/ListButtonController.swift:
```diff
@@ -0,0 +1,160 @@
+        picker.dataSource = self
+        header = tableView.dequeueReusableHeaderFooterView(withIdentifier: ListTableHeader.identifier) as? ListTableHeader
+        header.delegate = self
+        
```
в методы по смыслу

KamnevVladimir commented on Auctix/List/ListButtonController.swift:
```diff
@@ -0,0 +1,160 @@
+    
+    override func viewWillLayoutSubviews() {
+        super.viewWillLayoutSubviews()
+        tableView.frame = view.bounds
```
фрейм в viewDidLoad
Почитай про жизненный цикл UIViewController

KamnevVladimir commented on Auctix/List/ListButtonController.swift:
```diff
@@ -0,0 +1,160 @@
+    }
+}
+
+extension ListButtonTabViewController: UITableViewDataSource, UITableViewDelegate {
```
в два разных расширения

и для удоства
// MARK: - Table View Delegate
// MARK: - Table View Data Source

если в Edit выбрать Minimap, то там видны все эти комменты и по ним можно перемещаться

KamnevVladimir commented on Auctix/List/ListButtonController.swift:
```diff
@@ -0,0 +1,160 @@
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: ExhibitionTableViewCell.identifier, for: indexPath) as? ExhibitionTableViewCell else { return UITableViewCell()}
+        let exhibition = exhibitions[indexPath.row]
+        cell.configure(with: exhibition)
+        cell.backgroundColor = .clear
```
cell.backgroundColor = .clear

эта настройка должна быть в ячейке

KamnevVladimir commented on Auctix/List/ListButtonController.swift:
```diff
@@ -0,0 +1,160 @@
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let rootVC = TableProductsController()
+        navigationController?.pushViewController(rootVC, animated: true)
```
в метод

KamnevVladimir commented on Auctix/List/ListTableHeader.swift:
```diff
@@ -0,0 +1,52 @@
+        super.init(reuseIdentifier: reuseIdentifier)
+        contentView.backgroundColor = .white
+        contentView.addSubview(headerLabel)
+        headerLabel.text = "Name" + arrow
```
в методы

KamnevVladimir commented on Auctix/List/ListTableHeader.swift:
```diff
@@ -0,0 +1,52 @@
+    private func setupButtons() {
+        headerLabel.snp.makeConstraints { make in
+            make.centerX.centerY.equalTo(contentView)
+            make.bottom.equalTo(contentView)
```
два конфликтующих констрейнта

точно нужен centerY ?

KamnevVladimir commented on Auctix/LoadingScreenViewController.swift:
```diff
@@ -0,0 +1,69 @@
+class LoadingScreenViewController: UIViewController {
+
+    let logoLabel = UILabel()
+    let logoImg = UIImage(named: "Image2")
```
- на будущее

шоби прям по красотища, можно в каждом файле делать свои константы с картинками
Так глаголили мастера из всяких типо топ компаний, но я на практике не использовал. На личное усмотрение и любопытство

KamnevVladimir commented on Auctix/LoadingScreenViewController.swift:
```diff
@@ -0,0 +1,69 @@
+    let logoLabel = UILabel()
+    let logoImg = UIImage(named: "Image2")
+    let contentLogoImg = UIImageView()
+    let contentMod = UIView.ContentMode(rawValue: 0)
```
шо за штука?

ShagMichail replied:
А, пхахахх, сначала хотел сделать загрузочный экран через контроллер, но потом мне написало, что нельзя использовать кастовый контроллер для загрузочной страницы, поэтому сейчас этот контроллер вообще не используется

KamnevVladimir commented on Auctix/LoadingScreenViewController.swift:
```diff
@@ -0,0 +1,69 @@
+            contentLogoImg.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 12),
+            contentLogoImg.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: 12),
+
+            
```
отступ 1

KamnevVladimir commented on Auctix/LoadingScreenViewController.swift:
```diff
@@ -0,0 +1,69 @@
+    
+    func setupImgView(){
+        
+        //contentLogoImg.image = logoImg
```
отступы

KamnevVladimir commented on Auctix/Onboarding/OnboardingViewController.swift:
```diff
@@ -0,0 +1,123 @@
+    private let buttonContinue = UIButton()
+    private let buttonSignUp = UIButton()
+  
+    
```
отступы

KamnevVladimir commented on Auctix/TabBar/TabBarViewController.swift:
```diff
@@ -0,0 +1,93 @@
+        super.viewDidLoad()
+        
+        viewControllers = [homeVC, listVC, bidVC, accVC]
+        self.delegate = self
```
тоже в методы

KamnevVladimir commented on Auctix/TabBar/TabBarViewController.swift:
```diff
@@ -0,0 +1,93 @@
+        
+        UITabBar.appearance().tintColor = .honeyYellow
+        
+        //пассивный цвет
```
узнаЮ тебя, Миша )

Отступы не так круто делай

KamnevVladimir commented on Auctix/TableProducts/Cell + Data/Product.swift:
```diff
@@ -0,0 +1,18 @@
+    let id: String = UUID().uuidString
+    let productImg: UIImage
+    let title: String
+    let text_opis: String
```
нейминг

ShagMichail replied:
Пока не добавлял бд и при добавлении переделаю

KamnevVladimir commented on Auctix/TableProducts/Cell + Data/ProductCell.swift:
```diff
@@ -0,0 +1,140 @@
+
+class ProductCell: UITableViewCell {
+    
+    static let nib = UINib(nibName: String(describing: ExhibitionCell.self), bundle: nil)
```
в расширение

KamnevVladimir commented on Auctix/TableProducts/Cell + Data/ProductCell.swift:
```diff
@@ -0,0 +1,140 @@
+        nameProd.text = data.title
+        cost.text = data.cost + "$"
+        time.text = "1 work 3 tausent"
+        
```
лишний

ShagMichail replied:
Кто?

KamnevVladimir commented on Auctix/TableProducts/ProductViewController.swift:
```diff
@@ -0,0 +1,224 @@
+    var product: Product? {
+        didSet {
+            guard let product = product else {
+                return
```
это можно в configure вынести

KamnevVladimir commented on Auctix/TableProducts/ProductViewController.swift:
```diff
@@ -0,0 +1,224 @@
+    private let question = UILabel()
+    
+    private var priceArray = ["","",""]
+    private let screenWidth = UIScreen.main.bounds.width
```
в константы

KamnevVladimir commented on Auctix/TableProducts/ProductViewController.swift:
```diff
@@ -0,0 +1,224 @@
+        setupPrice()
+        
+        view.addSubview(titleLabel)
+        view.addSubview(productImageView)
```
в метод

KamnevVladimir commented on Auctix/TableProducts/ProductViewController.swift:
```diff
@@ -0,0 +1,224 @@
+    }
+    
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
```
Группируй заранее методы рядом

Вообще, вот структура правильная

class Controller {
1) Properties
2) Initialization
3) Life Cycle Methods
4) Methods

KamnevVladimir commented on Auctix/TableProducts/ProductViewController.swift:
```diff
@@ -0,0 +1,224 @@
+    func setupLayuot(){
+        NSLayoutConstraint.activate([
+            titleLabel.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            titleLabel.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
```
c pinLayout выйдет в несколько строк

KamnevVladimir commented on Auctix/TableProducts/TableProductsController.swift:
```diff
@@ -0,0 +1,105 @@
+        }
+        productViewController.dismiss(animated: true)
+        
+        let alertVC = UIAlertController(title: "Wow!", message: "Your bet has been placed", preferredStyle: .alert)
```
- обяз

Создайте фабрику алертов. Сделайте ее либо синглтоном, либо через статичные методы генерируете алерты. В методы передаете различный текст, в идеале, через протокол

ShagMichail committed:
Исправил недочеты ментора (не все)

ShagMichail committed:
Изменил Picker

IvanTuly committed:
Add login

ShagMichail committed:
Merge pull request #14 from Auctix/feature/ivan/login

Add login

ShagMichail committed:
Merge branch 'dev'

ShagMichail committed:
Исправил стиль спикера для продукта

ShagMichail committed:
Merge pull request #15 from Auctix/feature/micha/firebase

Feature/micha/firebase

bashun98 committed:
fixed Vova's comments

bashun98 committed:
Merge pull request #16 from Auctix/feature/eugene/fix

fixed Vova's comments

ShagMichail committed:
Первый ввод Firebase

ShagMichail committed:
Merge pull request #17 from Auctix/feature/micha/firebase

Первый ввод Firebase

ShagMichail committed:
Настройка регистрации и логина, добавление новой вью для перехода на логин

ShagMichail committed:
Запуск кнопки не удачен

ShagMichail committed:
Кнопка переключения на логин заработала

ShagMichail committed:
Настроил переход страниц для логина и регистрации

ShagMichail committed:
Сделал вторую вьюху для аккаунта

ShagMichail committed:
Изменил настройку страницы продукта и изменил настройку показа границы аккаунта

ShagMichail committed:
Исправил приветствие на странице аккаунта

ShagMichail committed:
Merge pull request #18 from Auctix/feature/micha/firebase

Feature/micha/firebase

ShagMichail committed:
Изменил ввод TextField

bashun98 committed:
fixed info.plist

bashun98 committed:
Merge pull request #19 from Auctix/feature/eugene/exhibitionDB

fixed info.plist

ShagMichail committed:
Добавляю алерт

ShagMichail committed:
Merge branch 'dev'

ShagMichail committed:
Сделал Алерт

ShagMichail committed:
Merge pull request #20 from Auctix/feature/micha/firebase

Feature/micha/firebase

ShagMichail committed:
Добавил подтверждение почты

ShagMichail committed:
Merge pull request #21 from Auctix/feature/micha/firebase

Добавил подтверждение почты

IvanTuly committed:
Add verification label in account page

IvanTuly committed:
Merge pull request #22 from Auctix/feature/ivan/verifingEmailLabel

Add verification label in account page

ShagMichail committed:
Первая попытка загрузки контроллера продуктов

ShagMichail committed:
Комментарии для Вовы

bashun98 committed:
DB for exhibitions without images

ShagMichail committed:
Изменение обновления страницы продукта

ShagMichail committed:
Merge branch 'dev'

ShagMichail committed:
Добавил работу лэйбла подтверждения почты на страницу аккаунта

ShagMichail committed:
Посмотреть для Вовы

bashun98 committed:
fixed

bashun98 committed:
Merge pull request #23 from Auctix/dev

Dev

bashun98 committed:
Merge pull request #24 from Auctix/feature/eugene/exhibitionDB

Feature/eugene/exhibition db

ShagMichail committed:
Обновление страницы продукта после разворачивания приложения

ShagMichail committed:
Исправил перезагрузку данных пользователя и также исправил добавление вью на странице аккаунта

ShagMichail committed:
Добавил сообщение на почту о смене пароля и пару лэйблов

ShagMichail committed:
Merge branch 'dev'

ShagMichail committed:
Merge pull request #25 from Auctix/feature/micha/firebase

Feature/micha/firebase

ShagMichail committed:
Добавил формат телефона на странице регистрации и тулбар для клавиатуры цифр

bashun98 committed:
Merge pull request #26 from Auctix/dev

Dev

ShagMichail committed:
Добавил даты для выставок и вывод выставок в домашнюю коллекцию в зависимости от даты

ShagMichail committed:
Добавил вывод количество дней до окончания выставки и поправил onCancle

ShagMichail committed:
Добавил добавление выставок из базы данных

ShagMichail committed:
Merge branch 'dev'

ShagMichail committed:
Merge pull request #27 from Auctix/feature/micha/firebase

Feature/micha/firebase

ShagMichail committed:
Сделал вывод продуктов относительно выставки (но только в коллекции)

ShagMichail committed:
Merge pull request #28 from Auctix/feature/misha/firebaseProduct

Сделал вывод продуктов относительно выставки (но только в коллекции)

bashun98 committed:
Merge pull request #29 from Auctix/dev

Dev

ShagMichail committed:
Вывод продуктов относительно выставок в List

ShagMichail committed:
Merge pull request #30 from Auctix/feature/misha/firebaseProduct

Вывод продуктов относительно выставок в List

bashun98 committed:
Merge branch 'dev'

ShagMichail committed:
Сделал, чтобы выделение ячеек не оставалось постоянным и изменил торги с продуктами. Теперь выводится цена, название и меняется цена, но пока локально

ShagMichail committed:
Сделал выполнение ставок относительно последнего поставившего пользователя.

ShagMichail committed:
Пытаюсь сделать обновление таблицы продуктов после повышения цены на товар

ShagMichail committed:
Merge pull request #31 from Auctix/feature/misha/firebaseProduct

Feature/misha/firebase product

bashun98 committed:
Merge pull request #32 from Auctix/dev

Dev

bashun98 committed:
add images for exh and prod

bashun98 committed:
Merge pull request #33 from Auctix/feature/eugene/exhibitionDB

Feature/eugene/exhibition db

bashun98 committed:
add back button for ListVC

ShagMichail committed:
Добавил страницу BidProduct

bashun98 committed:
add dispatch queue in exhtablewviewcell an vk logo in assets

bashun98 committed:
Merge pull request #34 from Auctix/feature/eugene/exhibitionDB

Feature/eugene/exhibition db

ShagMichail committed:
Merge branch 'dev'

Conflicts:
	Auctix/List/ExhibitionTableViewCell.swift

ShagMichail committed:
Сделал страницу Bid

ShagMichail committed:
Merge pull request #35 from Auctix/feature/misha/firebaseProduct

Feature/misha/firebase product

ShagMichail committed:
Поправил сортировку продуктов и добавил фоновые фотки

ShagMichail committed:
Сделал отступы для TextFild

ShagMichail committed:
Сделал чтобы TextFild на домашней страницу скрывался

ShagMichail committed:
Сделал размытие экрана при сворачивании приложения

ShagMichail committed:
Исправил работу страницы Bid

ShagMichail committed:
Merge pull request #36 from Auctix/feature/misha/fixError

Feature/misha/fix error

ShagMichail committed:
Исправил размер домашней коллекции

ShagMichail committed:
Merge pull request #37 from Auctix/feature/misha/fixError

Исправил размер домашней коллекции

ShagMichail committed:
Сново исправил домашнюю коллекцию

ShagMichail committed:
Начало работы с тем, чтобы клавиатура не закрывала TextFild

ShagMichail committed:
Сделал клавиатуру для контроллера продукта и понимания вью

ShagMichail committed:
Сделал поднятие View на контроллере регистрации, чтобы не закрывались поля ввода

ShagMichail committed:
Немного подогнал стиль и разобрался с алеутами на контроллере продуктов

ShagMichail committed:
Merge pull request #38 from Auctix/feature/misha/fixError

Feature/misha/fix error

ShagMichail committed:
Переделал контроллер Аккаунта

ShagMichail committed:
Поправил страницу Bid, сделал кнопку на аккаунте отсылки повторного письма подтверждения почты, сделал редактирование аккаунта(кроме фотографии)

ShagMichail committed:
Merge pull request #39 from Auctix/feature/misha/fixError

Feature/misha/fix error

ShagMichail committed:
Делаю коллекцию избранного на странице аккаунта

ShagMichail committed:
Коллекция на странице аккаунта появляется, но пока не настроено изменение ее при смене пользователя, также непонятно прыгает страница аккаунта (непонятное ее размещение на вью)

bashun98 committed:
add podfile

bashun98 committed:
add cache for photo

bashun98 committed:
add default image

ShagMichail committed:
Исправил недочеты стиля и сделал отслеживание коллекции на аккаунте

bashun98 committed:
add action for sighUP button in onboarding

bashun98 committed:
Merge pull request #40 from Auctix/feature/eugene/exhibitionDB

Feature/eugene/exhibition db

ShagMichail committed:
Merge branch 'dev'

ShagMichail committed:
Merge dev into fixError

ShagMichail committed:
Merge pull request #41 from Auctix/feature/misha/fixError

Feature/misha/fix error

ShagMichail committed:
Сделал, чтобы после первого запуска и регистрации перекидывало на TabBarController

ShagMichail committed:
Настроил страницу регистрации при первом входе и добавил кнопку регистрации позже

ShagMichail committed:
Сделал рабочую коллекцию на странице аккаунта

ShagMichail committed:
Merge pull request #42 from Auctix/feature/misha/fixError

Feature/misha/fix error

bashun98 committed:
.

bashun98 committed:
Merge branch 'dev'

ShagMichail committed:
Сделал загрузку фотографий в коллекцию на аккаунте

ShagMichail committed:
Merge pull request #43 from Auctix/feature/misha/fixError

Сделал загрузку фотографий в коллекцию на аккаунте

bashun98 committed:
add

bashun98 committed:
Merge branch 'dev'

Conflicts:
	Auctix/Account/Controller/AccountButtonController.swift

bashun98 committed:
Merge pull request #44 from Auctix/feature/eugene/exhibitionDB

Feature/eugene/exhibition db

ShagMichail committed:
Поменяли файл подов

ShagMichail committed:
Поменял файл подов

ShagMichail committed:
Исправил коллекцию аккаунта

bashun98 committed:
change google plist

ShagMichail committed:
Сделал регистрацию на странице продукта

ShagMichail committed:
Merge pull request #45 from Auctix/feature/misha/fixError

Feature/misha/fix error

bashun98 committed:
add simple local notifications in 5 days out + new pod for date

ShagMichail committed:
Сделал систему "лайков"

ShagMichail committed:
Сделал
1) статическую ориентацию экранов
2) зачеркивание текста цены на товаре, если выставка закрылась
3) изменение лейбла на контроллере продукта, если выставка продукта закрылась

ShagMichail committed:
Merge pull request #46 from Auctix/feature/misha/like

Feature/misha/like

bashun98 committed:
fixed toolbars

IvanTuly committed:
Account page

IvanTuly committed:
dark theme

IvanTuly committed:
login page update

IvanTuly committed:
Merge pull request #47 from Auctix/features/ivan/accountPage

Features/ivan/account page

bashun98 committed:
fixed textfield

bashun98 committed:
add NotificationService + fixed AppDelegate

bashun98 committed:
fixed

bashun98 committed:
fixed

bashun98 committed:
add map for ListVC

bashun98 committed:
fixed

bashun98 committed:
fixed

bashun98 committed:
fixed

bashun98 committed:
fixed

bashun98 committed:
Merge branch 'dev' into feaute/eugene/fixConstraints

# Conflicts:
#	Auctix/AppDelegate.swift
#	Auctix/List/ListTableHeader.swift
#	Auctix/List/ListViewController.swift

bashun98 committed:
.

bashun98 committed:
Merge pull request #49 from Auctix/feaute/eugene/fixConstraints

Feaute/eugene/fix constraints

ShagMichail committed:
Был добавлен поиск по продуктам, исправлена страница аккаунта, добавлены лайки на странице аккаунта, чтобы можно было убирать продукт из списка понравившихся

ShagMichail committed:
Исправил работу лайков, корректное отображение выставок

ShagMichail committed:
Сделал форму поддержки

ShagMichail committed:
Исправил работу лайков, добавил загрузку и выгрузку из БД фотографий пользователя

ShagMichail committed:
Создал кружок загрузки для фотографии

ShagMichail committed:
Merge pull request #50 from Auctix/feature/misha/search

Feature/misha/search
